winml/lib/Api/LearningModelSession.cpp

```diff
@@ -303,13 +200,14 @@ LearningModelSession::EvaluateFeaturesAsync(
   return EvaluateAsync(binding, correlation_id);
 }
 
-static onnxruntime::IOBinding&
+static _winmla::IIOBinding*
 GetIOBinding(
     winrt::com_ptr<winmlp::LearningModelBinding> binding_impl,
     winml::LearningModel& model) {
   // Get the IOBinding Collection, and bound outputs
-  auto& io_binding = binding_impl->BindingCollection();
-  auto& bound_output_names = io_binding.GetOutputNames();
+    com_ptr<_winmla::IIOBinding> io_binding;
+    io_binding.attach(binding_impl->BindingCollection());
```

Comment:
spacing

Reply:
good catch !

---
In reply to: [347042505](https://github.com/microsoft/onnxruntime/pull/2406#discussion_r347042505) [](ancestors = 347042505)